### PR TITLE
Trigger a new build of 0.16.3 as subclassing currently fails

### PR DESCRIPTION
**Pull request by Lnaden**
Objects in the current version of MongoEngine 0.16.3 on Conda-Forge cannot be subclassed in Python 3 due to a Python 2 dictionary `iteritems` method in `metaclass.py`

The following snippet is the minimal case to recreate the error from a fresh Conda install (with conda-forge channel as higher priority)
```bash
conda install --yes mongoengine
python
```

Then within Python

```python
from mongoengine import Document as D
class Foo(D):
    pass
```

throws the following:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "{conda_root}/envs/me/lib/python3.7/site-packages/mongoengine/base/metaclasses.py", line 330, in __new__
    meta.merge(base._meta)
  File "{conda_root}/envs/me/lib/python3.7/site-packages/mongoengine/base/metaclasses.py", line 441, in merge
    for k, v in new_options.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

However, doing a local build from cloning this repo, making no changes, and installing the local build allows the above script to work.
I think something went wrong during the dependency resolution when building this which would not have shown up in tests at the time (likely the `six` package).

Because a local build works fine, I think simply bumping the build version and allowing a new build to be pushed will resolve this issue (and will not remove old builds from the existing cloud)

@conda-forge-admin, please rerender

edit: fix block code formatting

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Lnaden** [comment]
@conda-forge/help-python 

Circle CI seems to be throwing an error I have not seen before, almost like the script for fetching the git heads are wrong? Its unclear though. I had originally made this PR through the GitHub edit of the file, which in turn made the fork and the branch automatically, so I thought that might have been it. I also tried making a branch the normal way and making a new PR, no avail. Its unclear to me what is throwing this error

https://circleci.com/gh/conda-forge/mongoengine-feedstock/30?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
#!/bin/bash -eo pipefail
./.circleci/fast_finish_ci_pr_build.sh
./.circleci/checkout_merge_commit.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

100    15  100    15    0     0    124      0 --:--:-- --:--:-- --:--:--   125
  File "<stdin>", line 1
    404: Not Found
       ^
SyntaxError: invalid syntax
Exited with code 1
```

**Comment by Lnaden** [comment]
@conda-forge/help-python

I think the issue might be that the 

@conda-forge-admin, please rerender 

command is not correctly updating the circle CI files from the 3.1.12, resulting in some odd errors. I would like to avoid manually editing the files if possible, any suggestions?

**Comment by Lnaden** [comment]
@conda-forge-admin, please rerender

Let see if breaking some files makes it re-render

**Comment by beckermr** [comment]
Download the repo and rerender manually.

**Comment by Lnaden** [comment]
Ah yes, the obvious answer I did not see! Worked like a charm, thank you @beckermr

**Comment by dgasmith** [comment]
Thanks @beckermr! Any chance for an admin to merge this in as this feedstock seems to be abandoned? Happy to be a maintainer here if desired.

**Comment by beckermr** [comment]
Ping @conda-forge/core for this.

**Comment by scopatz** [comment]
Please ping @conda-forge/mongoengine (or the maintainers directly). Also, I usually give it at least a week before pulling the "core" card and merging against the maintainers will.  Though I do shorten this to about about a day if the PR adds a maintainer :wink:

**Comment by dgasmith** [comment]
@Lnaden Lets give this a day and then add both of us as maintainers?

**Comment by Lnaden** [comment]
Sounds good

**Comment by Lnaden** [comment]
@conda-forge/mongoengine @bgruening @mariusvniekerk (pinging you all as previous PR mergers), does this look okay?

I've added @dgasmith and myself as two people who are willing to help maintain this feedstock as well.

**Comment by dgasmith** [comment]
Happy to be a maintainer here if confirmation is required.

**Comment by Lnaden** [comment]
@conda-forge/core I know its been a a short period, but we are at a point with a [new Conda Forge recipe](https://github.com/conda-forge/staged-recipes/pull/7784) where this PR will be needed to build correctly. As @dgasmith and I are both willing to help maintain this feedstock, can I request a merge?

**Comment by ocefpaf** [comment]
> Woo! Thank you!

Thanks you for picking up this feedstock.
